whois-nrtm4/src/main/java/net/ripe/db/nrtm4/RpslAttributeTransformer.java

```diff
@@ -0,0 +1,26 @@
+package net.ripe.db.nrtm4;
+
+import net.ripe.db.whois.common.rpsl.DummifierNrtm;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import org.springframework.stereotype.Component;
+
+
+/**
+ * Replaces some attribute values with dummy values to prevent leaking potentially sensitive information
+ */
+@Component
+public class RpslAttributeTransformer {
+
+    private final DummifierNrtm dummifierNrtm;
+
+    private final static int nrtmVersionNumber = 4;
+
+    RpslAttributeTransformer(final DummifierNrtm dummifierNrtm) {
+        this.dummifierNrtm = dummifierNrtm;
+    }
+
+    RpslObject filter(final RpslObject in) {
+        return dummifierNrtm.dummify(nrtmVersionNumber, in);
+    }
```

Comment:
Are we Dummyfying the data or filtering it? if its just one line I would rather call  it directly instead of creating a new class

Reply:
Agree with you, I guess it too much. if we need more logic in filtering we can revisit this and create a separate class but for now its best to call directly